src/extender.ts

```diff
@@ -1,64 +1,49 @@
-// @ts-ignore
-import * as gcl from "@google-cloud/logging";
-import { Extender, logger, field } from "./logger";
+import * as gcl from "@google-cloud/logging"
+import { Extender, field, Level, logger } from "./logger"
 
 export const createStackdriverExtender = (projectId: string, logId: string): Extender => {
-	enum GcpLogSeverity {
-		DEFAULT = 0,
-		DEBUG = 100,
-		INFO = 200,
-		NOTICE = 300,
-		WARNING = 400,
-		ERROR = 500,
-		CRITICAL = 600,
-		ALERT = 700,
-		EMERGENCY = 800,
-	}
+  const logging = new gcl.Logging({
+    autoRetry: true,
+    projectId,
+  })
 
-	const logging = new gcl.Logging({
-		autoRetry: true,
-		projectId,
-	});
+  const log = logging.log(logId)
+  const convertSeverity = (severity: Level): gcl.protos.google.logging.type.LogSeverity => {
```

Comment:
?: why do we do this? Or what is this rather?

Reply:
Our logger uses log levels 0-4 to represent trace, debug, info, warn, and error, but Google Cloud uses something like 800, 1000, etc. So we need to convert the level to the right number.

Reply:
I should also note logging to Google Cloud is optional.